sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/Trigger.java

```diff
@@ -263,13 +264,16 @@
     public abstract MergingTriggerInfo trigger();
   }
 
-  @Nullable
-  protected final List<Trigger> subTriggers;
+  public final List<Trigger> subTriggers;
```

Comment:
Done. I also should certainly have made an accessor; typo there. But it wasn't needed so I've made it protected and let each subclass expose what they need, as it will be when they are autovalue structs.